etcdserver/server.go

```diff
@@ -974,9 +976,9 @@ func (s *EtcdServer) monitorVersions() {
 
 func (s *EtcdServer) updateClusterVersion(ver string) {
 	if s.cluster.Version() == nil {
-		log.Printf("etcdsever: setting up the initial cluster version to %v", ver)
+		plog.Infof("setting up the initial cluster version to %v", ver)
 	} else {
-		log.Printf("etcdsever: updating the cluster version from %v to %v", s.cluster.Version(), ver)
+		plog.Infof("updating the cluster version from %v to %v", s.cluster.Version(), ver)
```

Comment:
after a more careful look, this should be a info rater than a notice. we should use notice when the version is updated.